### PR TITLE
Add "@uifabric/experiments" dependency to fabric-website

### PR DESCRIPTION
**Pull request by sopranopillow**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

(give an overview)

#### Focus areas to test

(optional)

### CODE REVIEW

File: apps/fabric-website/package.json [hunk 1]
```diff
@@ -38,8 +38,10 @@
     "write-file-webpack-plugin": "^4.1.0"
   },
   "dependencies": {
+
```
Comment (dzearing) [inline apps/fabric-website/package.json]: when you add a dependency, you should run `npm run generate` from the root to regen the shrinkwrap.

**Comment by sopranopillow** [comment]
@jordandrako I'm going to add the Foundation and the Collapsible section so that I don't have to use the ../../../ when importing them.

**Comment by sopranopillow** [comment]
@dzearing @kenotron I checked why I was getting the error on the build and I was able to fix it. The error shown when building utilities alone is the following:

error

which follows to this:

content

And I fixed it by doing this: (first image before and second after)

before

after

I was going to commit the change but I'm not getting the change when doing git status

File: common/config/rush/npm-shrinkwrap.json [hunk 2]
```diff
@@ -2650,64 +2653,6 @@
       "resolved": "https://registry.npmjs.org/@types/z-schema/-/z-schema-3.16.31.tgz",
       "integrity": "sha1-LrHQCl5Ow/pYx2r94S4YK2bcXBw="
     },
-    "@uifabric/charting": {
```
Comment (kenotron) [inline common/config/rush/npm-shrinkwrap.json]: Something seems *really wrong* here... the @uifabric packages seemed to have disappeared when you ran this... not sure what happened??

**Pull request merged**